autospotting.go

```diff
@@ -215,6 +217,8 @@ func (c *cfgData) parseCommandLineFlags() {
 	flag.StringVar(&c.LicenseType, "license", "evaluation", "\n\tControls the terms under which you use AutoSpotting"+
 		"Allowed values: evaluation|I_am_supporting_it_on_Patreon|I_contributed_to_development_within_the_last_year|I_built_it_from_source_code\n"+
 		"\tExample: ./AutoSpotting --license evaluation\n")
+	flag.StringVar(&c.BeanstalkCFNWrappers, "beanstalk_cfn_wrappers", "", "\n\tControls whether AutoSpotting patches Elastic Beanstalk UserData scripts to use the instance role when calling CloudFormation helpers instead of the standard CloudFormation authentication method\n"+
+		"\tExample: ./AutoSpotting --beanstalk_cfn_wrappers true\n")
```

Comment:
this needs to ne updated to `On`, unless we decide to keep the boolean values as they are as per my other comment